wwwapp/migrations/0059_reverse_userprofile_userinfo.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-03-08 10:24
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def forwards_func(apps, schema_editor):
+    UserProfile = apps.get_model("wwwapp", "UserProfile")
+    UserInfo = apps.get_model("wwwapp", "UserInfo")
+    db_alias = schema_editor.connection.alias
+
+    for user_info in UserInfo.objects.using(db_alias).all():
+        if not UserProfile.objects.filter(user_info__pk=user_info.pk).exists():
+            raise Exception(
+                'Orphaned UserInfo object found (pk=%d). Please verify and remove it manually first.' % user_info.pk)
```

Comment:
Should this be happening automatically or not?

Reply:
This is OK. Have you checked that Django performs reversing of previous two operations correctly if this exception is thrown?

Reply:
I'm 99% sure it does but I'll check

Reply:
I checked the Django docs - it does revert on Postgres, but does not on MySQL as it has no support for transactions around schema alterations (and SQLite support is just copy all data into a new table anyway so no problem here). Would you want me to move this check so that it happens before any changes are applied just in case?

Reply:
Please move it so the situation is super clear.

Reply:
Also it will be usefull to get the list of all pk-s of orphaned UserInfo objects. I think it would be best if you could supply a script to run in shell to get all orphaned user info ids